deeppavlov/models/morpho_tagger/common.py

```diff
@@ -56,7 +56,9 @@ def predict_with_model(config_path: [Path, str], infile: Optional[Union[Path, st
         else:
             data = sys.stdin.readlines()
     model = build_model(config, load_trained=True)
-    model.pipe[-1][-1].set_format_mode(output_format)
+    for elem in model.pipe:
+        if isinstance(elem[-1], TagOutputPrettifier):
+            elem[-1].set_format_mode(output_format)
     answers = model.batched_call(data, batch_size=batch_size)
     for elem in answers:
         print(elem)
```

Comment:
What about to remove print from here, as answers are returned by this function?

Reply:
You are right, done.

Reply:
unresolving, it is not done yet.

Reply:
Check https://github.com/deepmipt/DeepPavlov/pull/1077/commits/dd1aeb65b00ef0ef359cd1ad58dfaef47fd91042

Reply:
sry